pkgs/top-level/python-packages.nix

```diff
@@ -9,16 +9,13 @@
 { pkgs
 , stdenv
 , python
+, qt5
+, libsForQt5
```

Comment:
It looks like this updated *all* of python-packages to qt515 - I had to update inkcut to 515 to reflect this (#97971). 

It's possible that more of the changes in 22167ae45fb27fac1e6aec7d1bf6ce82516fcb12 can now be rolled back, but I tried a few and they don't seem to have the same problem inkcut had.

Reply:
That was not intended. Is it feasible to fix all python packages that were broken by this or should we change the python packages back to use Qt 5.14, while using pyqtwebengine with Qt 5.15 only for qutebrowser?

Reply:
I don't know, I tried a bunch of them and they all seemed fine (except for inkcut).

Personally I like the change and think we should fix any packages that might have been broken by this, but I guess it's hard to find out which are?

Reply:
Found this just now, maestral-gui is affected (https://github.com/NixOS/nixpkgs/issues/98067). Not sure what the correct fix for this is?